test/unit/data-generator.js

```diff
@@ -0,0 +1,361 @@
+var expect = require('chai').expect,
+    objectAssign = require('object-assign'),
+    generate = require('../../lib/data-generator.js'),
+    EventEmitter = require('events').EventEmitter;
+
+var root = {
+      title: ''
```

Comment:
hmm. I don't know...eslint says it's good, I have tried to reduce the identation, but it says:

>  7:5  error  Expected indentation of 6 space characters but found 4  indent
   8:3  error  Expected indentation of 4 space characters but found 2  indent
   9:3  error  Expected indentation of 4 space characters but found 2  indent
  12:3  error  Expected indentation of 4 space characters but found 2  indent
  17:3  error  Expected indentation of 4 space characters but found 2  indent
  22:3  error  Expected indentation of 4 space characters but found 2  indent
  27:3  error  Expected indentation of 4 space characters but found 2  indent
  32:3  error  Expected indentation of 4 space characters but found 2  indent
  37:3  error  Expected indentation of 4 space characters but found 2  indent